features/openhab-addons/src/main/feature/feature.xml

```diff
@@ -164,6 +164,11 @@
         <bundle start-level="80">mvn:org.openhab.binding/org.openhab.binding.keba/${project.version}</bundle>
     </feature>
 
+    <feature name="openhab-binding-knx" description="KNX Binding" version="${project.version}">
+        <feature>openhab-runtime-base</feature>
```

Comment:
I wonder if we should also add the serial transport feature here as otherwise people with an FT1.2 will wonder why it isn't working? Having RXTX installed for all people with an IP interface should not do too much harm...